tests/test_callbacks.py

```diff
@@ -4,4 +4,4 @@ def test_when_imagej_starts(ij):
     when_imagej_starts, which injects a small piece of data into the gateway
     object. We check for that data here to make sure the callback happened.
     """
-    assert "success" == getattr(ij, "_when_imagej_starts_result", None)
```

Comment:
Does `assert ij._testing is True` not work? 😅

Reply:
It does, but it upsets `flake8`!

Reply:
Gotcha, well it's not tooooooo bad as is, so let's just leave it like this!